site/deploy/about/about.md

```diff
@@ -34,6 +34,23 @@ The Berkeley Center for Law and Technology supported this research in part in co
 
 This work was supported by the European Union’s Seventh Framework Programme (FP7), via the [PRIPARE project](http://pripareproject.eu/).
 
+## Attribution
```

Comment:
I see that `site/content/about/about.md` is indeed missing the new team text. This is also the case in [privacypatterns master](https://github.com/privacypatterns/privacypatterns/blob/master/site/content/about/about.md). Is it possible that was only added then to the already built copy-over (`site/deploy/about/about.md` )? In that case shall we just replace `content...about.md` with `deploy...about.md` in this PR?